core/src/main/resources/config.yml

```diff
@@ -223,8 +227,13 @@ autoLoginFloodgate: false
 # Enabling this might lead to people gaining unauthorized access to other's accounts!
 allowFloodgateNameConflict: false
 
-# This enables auto registering every player connecting through Floodgate.
-# autoLoginFloodgate must be 'true' for this to work
+# Automatically register players connecting through Floodgate.
+# autoLoginFloodgate must be available for the player to use this
+# Possible values:
+#   false: Disables auto registering for every player connecting through Floodgate
+#   true: Enables auto registering for every player connecting through Floodgate
+#   no-conflict: Bedrock players will only be automatically registered if no Premium Java MC account exists
```

Comment:
Please state that no-conflicts performs a possible rate-limiting check against Mojang.

Reply:
Rephrased the sentence to include that & rebased the commit.